incent-config/Dangerfile

```diff
@@ -133,8 +133,8 @@ def branch_contains_prefix?
   prefixes.any? { |prefix| branch_name.start_with?(prefix) }
 end
 
-def branch_name_contains_underscore?
-  @branch_name_contains_underscore ||= branch_name.include?('_')
+def branch_name_contains_no_underscore?
+  @branch_name_contains_underscore ||= !branch_name.include?('_')
```

Comment:
This should be `@branch_name_contains_no_underscore`

Reply:
good catch!